tasks.py

```diff
@@ -1120,6 +1117,28 @@ def ci_build(context):
     lock_f.close()
 
 
+@task
+def install_cloudrendering_engine(context, force=False):
+    if not sys.platform.startswith("darwin"):
+        raise Exception("CloudRendering Engine can only be installed on Mac")
+    s3 = boto3.resource("s3")
+    target_base_dir = "/Applications/Unity/Hub/Editor/{}/PlaybackEngines".format(_unity_version())
```

Comment:
Perhaps using the `_unity_path()` function here to not have multiple places to change if this string changes.